.vscodeignore

```diff
@@ -1,3 +1,6 @@
 .github/**
+.gitmodules
 .gitignore
 .vscode/**
+motoko/**
+node_modules/**
```

Comment:
this brings down the `.vsix` size from 44MB to 7.3MB :-)

Reply:
we should do it on the `master` branch too!

Reply:
Are there any other files we could ignore for packaging?

Reply:
Probably! I was thinking of using [Rollup](https://rollupjs.org/guide/en/) to tree-shake and bundle the TS code, but it seems like this also works.